src/main/java/lotto/domain/LottoMatcher.java

```diff
@@ -41,11 +41,20 @@ public int price() {
         return price;
     }
 
-    public void incrementScore(LottoScore lottoScore){
-        lottoScoreConsumer.accept(lottoScore);
+    public void incrementScore(Map<String, Integer> map) {
+        map.putIfAbsent(mapKey, 0);
+        map.computeIfPresent(mapKey, (k, v) -> v + 1);
```

Comment:
LottoScore에서 이미 0으로 초기화를 하고 있으니 putIfAbsent는 없어도 될 것 같아요 😃 
```suggestion
        map.computeIfPresent(mapKey, (k, v) -> v + 1);
```